libc/src/__support/FPUtil/FPBits.h

```diff
@@ -501,6 +501,10 @@ struct FPRepSem<FPType::X86_Binary80, RetT>
                        Significand::msb() | (Significand::msb() >> 1) |
                            Significand(v)));
   }
+  LIBC_INLINE static constexpr RetT get_canonical_val(Sign sign = Sign::POS,
+                                                      StorageType v = 0) {
+    return RetT(encode(sign, Exponent::min(), Significand(v)));
+  }
```

Comment:
Remove this function, use `make_value` instead.

Reply:
done.